11LessonEleven/in-class-project/css/style.css

```diff
@@ -0,0 +1,145 @@
+header{
+  background: #2BCDC0;
+  width: 100%;
+  height:400px;
+  color:#F7F5F6;
+}
+
+header h1{
+  padding:15%;
+  text-align: center;
+  font-size: 40px;
+}
+
+.clearfix:after{
+  content:"";
+  clear: both;
+  display: block;
+}
+
+.clearfix{
+  background-color: #1B1B1B;
+  width:100%;
+  text-align: center;
+}
+
+.babyblue{
+float:left;
+background: #1B1B1B;
+color: #F7F5F6;
+height: 200px;
+width:200px;
+padding-top: 50px;
+padding-left: 250px;
+padding-right: 50px;
+}
+
+.blueborder{
+  background:#33A8DD;
+  width: 100%;
+  height: 30px;
+}
+
+.orange{
+  float:left;
+  background: #1B1B1B;
+  color: #F7F5F6;
+  height: 200px;
+  width: 200px;
+  padding: 50px;
+}
+
+.orangeborder{
+  background:#F66530;
+  width: 100%;
+  height: 30px;
+}
+
+.teal{
+  float:left;
+  background: #1B1B1B;
+  color: #F7F5F6;
+  height: 200px;
+  width: 200px;
+  padding: 50px;
+}
+
+.tealbox{
+  background:#2BCDC0;
+  width: 100%;
+  height: 30px;
+}
+
+h2{
+  font-size: 30px;
+  text-align: center;
+  padding-bottom: 30px;
+}
+
+@media (max-width: 768px) {
+
+header{
+  background-color: #33A8DD;
+  color: white;
+  width: 100%;
+  height:250px;
+  text-align: left;
+}
+
+.clearfix{
```

Comment:
I would rather you leave the `.clearfix` class just for the purpose of fixing parent collapse and use another selector for that element